test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -176,16 +175,16 @@ def run_data_service_test(ec2_connection, tensorflow_training):
 @pytest.mark.model("N/A")
 @pytest.mark.parametrize("ec2_instance_type", TF_EC2_CPU_INSTANCE_TYPE, indirect=True)
 def test_tensorflow_dataservice_cpu(tensorflow_training, ec2_connection, cpu_only):
-	if is_tf1(tensorflow_training):
-		pytest.skip("This test is for TF2 only")
-	run_data_service_test(ec2_connection, tensorflow_training)
+    if not is_tf23(tensorflow_training):
```

Comment:
This needs to be forward thinking. It should only check if its less than tf23. tf24 might have the same API.

Reply:
updated with above_tf23